### DISCUSSION TITLE
Eslint 9.0.0 does not work

### DISCUSSION THREAD

**Discussion by gabriel-logan** [eslint/eslint#18307] [General]
In several projects I spent hours trying to find the problem and saw that it was the eslint of the new version, simply by default npm downloads the latest version that is on NPM, then it downloads version 9.0.0 and this version may even be working, but then it is incompatible with all previous eslint configurations, etc., almost every parser or config uses old versions of eslint, SOLUTION: whenever I downgrade from 9.0.0 to eslint@8.57.0 everything works again.

**Comment by Rec0iL99** [comment]
Hi @gabriel-logan, by default `v9.0.0` uses the new flat config system and the `.eslintrc` config system is now deprecated. If you want to continue using `v9.0.0` with the `.eslintrc` config, then you must set the environment variable `ESLINT_USE_FLAT_CONFIG ` to `false`.

**Reply by gabriel-logan** [reply]
Thanks.

Its the .dev file ?
